### DISCUSSION TITLE
크롬 웹스토어에 검색이 안되는데 혹시 내려갔나요?

### DISCUSSION THREAD

**Discussion by MKK425** [changdoc/naver-vaccine-macro#89] [Q&A]
이제서야 알게되어 설치하려고했는데 검색하니까 나오질 않네요 ㅠㅠ

**Comment by changdoc** [comment]
네 버전 패치가 안되어 신규 버전은 배포를 못하고 있는 상황이고요.
기존에 공개되어 있는 버전은 동작하지 않고 계정이 차단되는 이슈가 있어 내려뒀습니다.

**Reply by MKK425** [reply]
어찌저찌해서 1.09버전?을 설치했는데 이 버전은 동작이 안되는건가요?
여기저기 뒤져가며 따라해서 병원목록까지 따는 것 까진했습니다.

**Reply by changdoc** [reply]
동작은 되나 새로고침 간격이 2초라 예약 성공 확률이 좀 떨어지는것 같네요

**Reply by choiGwanggyun** [reply]
어디서 다운 받을수 있을까요?
